features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/AdminPages/Addons/InputListBase.jsx

```diff
@@ -56,13 +56,13 @@ const InputList = (props) => {
             setId(i);
         } else {
             setId(id + 1);
-            setUserInputItems([{ key: '' + id, value: '', error: onValidation('') }]);
+            setUserInputItems([{ key: '' + id, value: '', error: onValidation(undefined) }]);
         }
     }, []);
 
     const onAddInputField = () => {
         setId(id + 1);
-        const newUserItemList = [...userInputItems, { key: '' + id, value: '', error: onValidation('') }];
+        const newUserItemList = [...userInputItems, { key: '' + id, value: '', error: onValidation(undefined) }];
```

Comment:
shall we avoid passing using these `undefined` values here? and other places as well

Reply:
Fixed with commit: https://github.com/wso2/carbon-apimgt/pull/8838/commits/2455d6596ee3933b261b5d8897ae2f37e49fd66e